index.html

```diff
@@ -0,0 +1,269 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Sofia">
+    <link rel="stylesheet" href="style.css">
+    <title>My Personal Portifolio</title>
+</head>
+<body>
+   <header class="header-container">
+
+        <nav class="navbar">
+
+            <a href="#"><img class="my-logo" src="images/Cizzla_logo.png" alt=""></a>
+
+
+            <ul class="nav-links">
+                <li><a class="links" href="#">Portifolio</a></li>
+                <li><a class="links" href="#">About</a></li>
+                <li><a class="links" href="#">Contact</a></li>
+                <li><a class="links" href="#"><img src="images/Icon - Mail.svg" alt=""></a></li>
+            </ul>
+
+                <i class="fas fa-bars"></i>
+        </nav>
+
+        <main class="main-container">
+
+            <ul class="social-icons">
+                <li><a href=""><img class="icons" src="images/github@2x.png" alt=""></a></li>
+                <li><a href=""><img class="icons" src="images/Linkedin icon@2x.png" alt=""></a></li>
+                <li><a href=""><img class="icons" src="images/angel list@2x.png" alt=""></a></li>
+                <li><a href=""><img class="icons" src="images/twitter@2x.png" alt=""></a></li>
+                <li><a href=""><img class="icons" src="images/medium@2x.png" alt=""></a></li>
+
+            </ul>
+
+            <div class="main-info">
+                <h1 class="info-header">Hey There.<br>I'm Ian.</h1>
+                <p class="info-title">I'm a Software Developer</p>
+                <p class="info-details">I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hesitate to contact me.</p>
+
+            </div>
+
+        </main>
+   </header>
+
+   <!-- Recent Work Section-->
+   <section class="work-container">
+
+        <div class="project-title">
+            <div class="header-title">
+                <h1>My Recent Works</h1>
```

Comment:
- Kindly you will have one `h1` element per page, so use other heading elements for this, I think `h2` would be better for better semantic arrangement 👍